### ISSUE TITLE
D417: false positive with *args/**kwargs in numpy docstrings

### ISSUE THREAD

**Issue by tlambert03** [charliermarsh/ruff#923]
In the [specification for numpy docstrings](https://numpydoc.readthedocs.io/en/latest/format.html#parameters), they state:

> When documenting variable length positional, or keyword arguments, leave the leading star(s) in front of the name:
> 
> ```
> *args : tuple
>     Additional arguments should be passed as keyword arguments
> **kwargs : dict, optional
>     Extra arguments to `metric`: refer to each metric documentation for a
>     list of all possible arguments.
> ```

but if you do that, ruff is unhappy:

```python
def f(*stuff: int):
    """Summary.

    Parameters
    ----------
    *stuff : int
        all the stuff.
    """
```
```
Missing argument description in the docstring: `stuff` Ruff(D417)
```

(if you remove the star in the docstrings, the error goes away)

**Comment by charliermarsh** [comment]
Oh interesting, thanks! Will fix.

**Comment by charliermarsh** [comment]
This is going out in [v0.0.142](https://github.com/charliermarsh/ruff/releases/tag/v0.0.142).

**Comment by tlambert03** [comment]
So fast :). Thanks again!
